test/unit/utils/shared_mocks.mocks.dart

```diff
@@ -50,7 +49,7 @@ class _FakeLinkedInProfileEmail extends _i1.Fake
 
 class _FakeResponse extends _i1.Fake implements _i10.Response {}
 
-class _FakeUint8List extends _i1.Fake implements _i11.Uint8List {}
+// class _FakeUint8List extends _i1.Fake implements _i11.Uint8List {}
```

Comment:
please revert changes inside this file, as you shouldn't modify this file by hand